liana/plotting/_dotplot.py

```diff
@@ -94,11 +96,20 @@ def dotplot(adata: anndata.AnnData = None,
             how = 'min'
         else:
             how = 'max'
-        top_lrs = _aggregate_scores(liana_res, what=orderby, how=how,
+
+        if met:
+            top_lrs = _aggregate_scores(liana_res, what=orderby, how=how,
+                                    entities=['interaction',
```

Comment:
I assume these are just temporary names, but they should be consistent with the ones in liana

Reply:
aggregate function needed for metalinks ?

Reply:
but if they are not complexes ?